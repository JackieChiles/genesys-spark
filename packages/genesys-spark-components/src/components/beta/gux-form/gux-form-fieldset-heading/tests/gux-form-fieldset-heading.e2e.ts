import { a11yCheck, newSparkE2EPage } from '../../../../../test/e2eTestUtils';

describe('gux-form-fieldset-heading', () => {
  describe('#render', () => {
    [
      `<gux-form-fieldset-heading><h2 slot="heading">Form Fieldset Header</h2></gux-form-fieldset-heading>`,
      `<gux-form-fieldset-heading><h3 slot="heading">Form Fieldset Header</h3></gux-form-fieldset-heading>`
    ].forEach((html, index) => {
      it(`should render the gux-form-fieldset component as expected (${index + 1})`, async () => {
        const page = await newSparkE2EPage({ html });
        const element = await page.find('gux-form-fieldset-heading');

        await a11yCheck(page);

        expect(element.outerHTML).toMatchSnapshot();
      });
    });
  });
});
