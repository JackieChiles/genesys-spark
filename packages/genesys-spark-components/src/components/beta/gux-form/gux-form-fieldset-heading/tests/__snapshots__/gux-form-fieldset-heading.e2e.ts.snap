// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-form-fieldset-heading #render should render the gux-form-fieldset component as expected (1) 1`] = `"<gux-form-fieldset-heading hydrated=""><h2 slot="heading">Form Fieldset Header</h2></gux-form-fieldset-heading>"`;

exports[`gux-form-fieldset-heading #render should render the gux-form-fieldset component as expected (2) 1`] = `"<gux-form-fieldset-heading hydrated=""><h3 slot="heading">Form Fieldset Header</h3></gux-form-fieldset-heading>"`;
